lib/org/openpsa/invoices/handler/invoice/view.php

```diff
@@ -55,6 +55,7 @@ private function load_datamanager() : datamanager
 
         if ($this->_config->get('invoice_pdfbuilder_class')) {
             $schemadb->get('default')->get_field('pdf_file')['hidden'] = false;
+            $schemadb->get('default')->get_field('pdf_file_reminder')['hidden'] = false;
```

Comment:
I don't see nything that adds this field to the dm schema

Reply:
also, this should be conditional on invoice_pdfbuilder_class_reminder, not invoice_pdfbuilder_class